modules/dnn/src/layers/gemm_layer.cpp

```diff
@@ -0,0 +1,197 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "../precomp.hpp"
+#include "layers_common.hpp"
+
+#include <opencv2/dnn/shape_utils.hpp>
+#include "cpu_kernels/fast_gemm.hpp"
+
+namespace cv { namespace dnn {
+
+class GemmLayerImpl CV_FINAL : public GemmLayer {
+public:
+    GemmLayerImpl(const LayerParams& params) {
+        setParamsFrom(params);
+
+        trans_a = params.get<bool>("transA", false);
+        trans_b = params.get<bool>("transB", false);
+        alpha = params.get<float>("alpha", 1.0f);
+        beta = params.get<float>("beta", 1.0f);
+
+        const_B = params.get<bool>("constB", false);
+        const_C = params.get<bool>("constC", false);
```

Comment:
Am I right that `C` is a bias? Can you please put a formula `A * B + C` from https://github.com/opencv/opencv/pull/23897/files#diff-f9fc55b2657f441025bc0adfeb683491b1e3d8275702e159efcb368a1b2c0dd5L1950-L1951 somewhere?

Reply:
> Am I right that C is a bias?

Yes.

> Can you please put a formula A * B + C

I put`Y = A * B + C` above `forward()` in the latest commit.